app/models/form.rb

```diff
@@ -75,7 +75,7 @@ def self.receivable_association
 
   after_save :update_minimum
 
-  validates :name, presence: true, length: {maximum: 32}
+  validates :name, presence: true, length: {maximum: 32}, format: {with: /\A[A-Za-z0-9]+\z/}
```

Comment:
Can we also allow space, dash, underscore? Those should ideally be safe. I also wonder if this is too restrictive because it will disallow any accented characters like é which should ideally also be safe.

Have you already checked with Dottie to see which form exactly is breaking the API, and what the error is? It may actually be a very specific kind of symbol that does it, such as a slash... I know Zied has reported that specific symbol in the past.